### ISSUE TITLE
When confirmation modal is opened,  make background elements are not accessible by keyboard 

### ISSUE THREAD

**Issue by stanleychh** [SAP/luigi#2411]
This is a follow-up ticket from #2377.

Make sure the background elements (left nav sidebar, top shellbar, etc.) are not accessible by keyboard (`Tab` key) when confirmation modal is opened.
